Description: Call the cleanup routine every minute, instead of twice per minute, to reduce wakeups and therefore power usage.
Author: Colin King <colin.king@ubuntu.com>

Index: gconf-3.2.6/gconf/gconfd.c
===================================================================
--- gconf-3.2.6.orig/gconf/gconfd.c	2014-03-13 13:03:04.000000000 +0000
+++ gconf-3.2.6/gconf/gconfd.c	2014-03-13 13:09:03.521886058 +0000
@@ -1121,7 +1121,7 @@
 
   if (main_loops == NULL)
     {
-      gulong timeout_len = 60*0.5; /* 60 s/min * .5 min */
+      gulong timeout_len = 60.0; /* 60 seconds */
       
       g_assert(timeout_id == 0);
       timeout_id = g_timeout_add_seconds (timeout_len,
